engine/src/main/java/org/terasology/input/internal/BindCommands.java

```diff
@@ -58,7 +58,6 @@ public String bindKey(@CommandParam("key") String key, @CommandParam("function")
         AZERTY.put(KeyId.Z, new SimpleUri("engine:forwards"));
         AZERTY.put(KeyId.S, new SimpleUri("engine:backwards"));
         AZERTY.put(KeyId.Q, new SimpleUri("engine:left"));
-        AZERTY.put(KeyId.A, new SimpleUri("Core:dropItem"));
```

Comment:
I think the whole special handling for different keyboard layouts is "broken" in the sense that it does not work with modules at all... we might just remove it at this point... 🤷

Reply:
Yeah the azerty and the one (or two?) other keyboard layout commands we have might be good examples of the sort of code we took in from _one_ contributor who found it useful at some point in the past, but it may not be worth maintaining as-is if we can gain some benefit from cleaning up and later re-adding if there's a better way to do it and more interest in using it.

Reply:
how would we want to handle different keyboard layouts? I also don't really like how we use annotations to register bindings against a whole class. Inventory has 15 or so classes just dedicated to keybindings. Probably need to look at how bindings work and also look at maybe how some other engines handle bindings and what we can do to make the process better. Some key press events happen depending on context so different events can be handled by the same key depending on what the player is doing. probably a way to handle bindings depending on conditions in the engine.

Reply:
I am a person who was glad to see the button for switching to a dvorak keyboard layout in the config screen! I just say that as +1 to interest for Cervator's "does anyone use this?" point, but I know it does nothing to address your concerns about its implementation.

(I have _ambitions_ for input configuration stuff myself, but I won't get to act on them for a while and this PR is not the place.)

Reply:
Feedback appreciated @keturn - would indeed love to see somebody actually using one of those layouts help make the implementation a little better 😁